prtpy/partitioning/recursive_number_partitioning_sy_v2.py

```diff
@@ -0,0 +1,190 @@
+"""
```

Comment:
The file should have a more meaningful name, such as "recursive_number_partitioning_moffitt"